cpp/src/Slice/Parser.cpp

```diff
@@ -1642,7 +1653,6 @@ SequencePtr
 Slice::Container::createSequence(const string& name, const TypePtr& type, MetadataList metadata)
 {
     SequencePtr p = make_shared<Sequence>(shared_from_this(), name, type, std::move(metadata));
-    _contents.push_back(p);
```

Comment:
I moved these to the bottoms of their respective `createXXX` methods for consistency of placement.
It doesn't meaningfully change anything.